Services/AuthenticationService/Controllers/AuthenticationController.cs

```diff
@@ -21,6 +29,8 @@ public class AuthenticationController : ControllerBase
         [HttpPost]
         public async Task<UserToken> Login([FromServices] ILoginCommand command, [FromBody] LoginRequest request)
         {
+            _logger.LogInformation("Success");

```

Comment:
Why do we right success before command will be execute?

Reply:
It`s just example